README.md

```diff
@@ -41,9 +41,9 @@ The repository has the following permanent branches:
 ![Settings Screen](Screenshots/Settings%20Screen.png)|![About Page](Screenshots/About%20Page.png)|![Change Password Page](Screenshots/Change%20Password%20Page.png)|
 ## Contact
 
-You can reach maintainers team @mentorship-team on [Systers Open Source Slack](http://systers.io/slack-systers-opensource/) or @systers/maintainers-mentorship-android on GitHub.
+You can reach our community at [AnitaB.org Open Source Zulip] (https://anitab-org.zulipchat.com/).
```

Comment:
```suggestion
You can reach our community at [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).
```
The markdown was not working properly, with that space

Reply:
@isabelcosta  done :)